appveyor.yml

```diff
@@ -99,6 +99,7 @@ test_script:
       --flag cardano-sl:with-wallet
       --flag cardano-sl-core:-asserts
       --flag cardano-sl-core:-dev-mode
+      --flag cardano-sl:bench
```

Comment:
I believe we had some problems with appveyor with benchs off. But @rockbmb, please try disabling it, maybe problems don't exist any more.